change/@fluentui-react-dialog-f53ccc8c-8b2f-4726-8d7a-e61990a04769.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "dialog title now takes whole row when there are no actions. This fixes an earlier regresssion.",
```

Comment:
```suggestion
  "comment": "fix: dialog title now takes whole row when there are no actions. This fixes an earlier regresssion.",
```

Reply:
I have incorporated the suggestion. Is this the expected format for change file comments?